app/models/spree/digital_link.rb

```diff
@@ -34,12 +33,20 @@ def reset!
       update_column :created_at, Time.now
     end
 
+    def attachment
+      if digital.drm?
+        digital.drm_records.find_by(line_item: line_item)&.attachment
```

Comment:
Why safe navigation? If there's a link, there should always be a line item and a drm record. I think it should even be `find_by!`. Or am I missing some case? (The other branch also doesn't have this conditional)